app/src/main/java/org/wikipedia/gallery/MediaDownloadReceiver.java

```diff
@@ -38,7 +38,7 @@ public void setCallback(@Nullable Callback callback) {
     public void download(@NonNull Context context, @NonNull FeaturedImage featuredImage) {
         String filename = FileUtil.sanitizeFileName(featuredImage.title());
         String targetDirectory = Environment.DIRECTORY_PICTURES;
-        performDownloadRequest(context, Uri.parse(featuredImage.getOriginal().getSource()), targetDirectory, filename, null);
+        performDownloadRequest(context, Uri.parse(featuredImage.getOriginal().getSource()), targetDirectory, filename, null, DownloadManager.Request.VISIBILITY_VISIBLE);
```

Comment:
I'm not sure I see the benefit of this. Why wouldn't we always want to use `VISIBILITY_VISIBLE_NOTIFY_COMPLETED`, which shows the notification during downloading as well as when it's completed?  This change will no longer show the notification when it's complete.